rust/noosphere-core/src/data/headers/version.rs

```diff
@@ -1,8 +1,11 @@
 use anyhow::anyhow;
 use std::{convert::Infallible, fmt::Display, str::FromStr};
 
+/// The Noosphere protocol version
```

Comment:
How does this correlate with API version e.g. `v0alpha2`?

Reply:
Mmm, good point. I guess we would typically want to bump the protocol version for this change. I'll have to think about that.

Reply:
I'm going to say for now that this version concerns the structure of the data, and is at a different layer from the transport (e.g., Gateway REST API). But, let's discuss this more in detail. https://github.com/subconsciousnetwork/noosphere/issues/625